pkgs/development/tools/shadered/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, sfml
+, glm
+, python3
+, glew
+, pkg-config
+, SDL2 }:
+
+stdenv.mkDerivation rec {
+  pname = "SHADERed";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "dfranx";
+    repo = "${pname}";
+    rev = "v${version}";
+    fetchSubmodules = true;
```

Comment:
I think we should use the system libraries instead the vendored ones.

Reply:
I would agree, but shadered depends on really specific versions of those libraries, so to keep it working it would prefer to keep it this way